bedrock/firefox/templates/firefox/facebookcontainer/index.html

```diff
@@ -105,7 +105,7 @@ <h3 class="mzp-c-card-title">{{ _('About Firefox and Mozilla') }}</h3>
       class='mzp-t-dark',
       heading_level=2)
     %}
-    {{ download_firefox(alt_copy=_('Download Now'), locale_in_transition=true, dom_id="download-firefox-today") }}
```

Comment:
Did you mean to remove `locale_in_transition`?

Reply:
Yup I did, this param shouldn't be here as the p[age uses a different lang file to /download/thanks/ 👍